.github/workflows/repo-sync.yml

```diff
@@ -6,6 +6,7 @@ on:
         - closed
       branches:
         - current
+        - equuleus
```

Comment:
But we have it in equuleus branch https://github.com/vyos/vyos-1x/blob/a79b59070ca0039aec0a43a68e427b1d75b40cd0/.github/workflows/repo-sync.yml#L8

Reply:
> But we have it in equuleus branch
> 
> https://github.com/vyos/vyos-1x/blob/a79b59070ca0039aec0a43a68e427b1d75b40cd0/.github/workflows/repo-sync.yml#L8

added backports for this branch

Also removed crux backport